src/index.js

```diff
@@ -26,4 +26,11 @@ naclFactory.instantiate(nacl => {
 	naclInstance = nacl;
 });
 
-export default { cryptography, transaction, API, time, passphrase };
+export default {
+	APIClient,
+	apiUtils: { getMainnetClient, getTestnetClient },
```

Comment:
We should be consistent with the pattern used in transactions for utils.

Reply:
do u mean changing `apiUtils` to some other name?

Reply:
I mean in `transactions` we have a `.utils` property for accessing transactions utils.